src/interfaces/message.h

```diff
@@ -29,6 +29,7 @@ struct Message {
  * and boundaries and avoid value clashing
  */
 enum : MessageId {
+  ArduinoStandardMessageStartId     = 0x000,
   ArduinoIOTCloudStartMessageId     = 0x100,
   ArduinoProvisioningStartMessageId = 0x200,
```

Comment:
```suggestion
  ArduinoMessageStartId             = 0x000,
  ArduinoIOTCloudStartMessageId     = 0x100,
  ArduinoProvisioningStartMessageId = 0x200,
  ArduinoVersionsStartId            = 0x300,
```

Reply:
I prefer keeping the versions messages IDs starting from ArduinoStandardMessageStartId. My idea is that the StandardMessageEncoder is a collection of common encoders

Reply:
A Concrete Message specifies the model data is going to take with that id. A Standard Message in my opinion is just a standard model a message may take, independently of its id. A `WifiVersionMessage` is the concrete usage of `StandardVersionMessage`, which represents its data model that may be shared with other concrete messages, i.e. `ProvisioningVersionMessage`, which will take another id, but the model is the same. 

For this reason I don't see the use for defining `ArduinoStandardMessageStartId`. Instead we can either take a id from `ArduinoIOTCloudStartMessageId` or `ArduinoProvisioningStartMessageId` depending on the library it is using that model, when instantiating the encoder.